arm-mobileengagement/2014-12-01/swagger/mobile-engagement.json

```diff
@@ -1289,6 +1289,548 @@
+                        "description": "Success, export jobs are returned as JSON.",
+                        "schema": {
+                            "$ref": "#/definitions/ExportJobResultList"
+                        }
+                    },
+                    "default": {
+                        "description": "* `400`: Invalid parameters or application is disabled. Check response body for details.\n* `401`: Authentication error.",
+                        "schema": {
+                            "$ref": "#/definitions/ApiError"
+                        }
+                    }
+                },
+                "x-ms-pageable": {
+                    "nextLinkName": "nextLink"
+                }
+            }
+        },
+        "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.MobileEngagement/appcollections/{appCollection}/apps/{appName}/devices/historicalExportJobs": {
+            "get": {
+                "tags": [
+                    "Export",
+                    "List"
+                ],
+                "operationId": "Devices_ListHistoricalExportJobs",
+                "description": "Get the list of export jobs.",
+                "parameters": [
+                    {
+                        "$ref": "#/parameters/SubscriptionIdParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ResourceGroupNameParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/AppCollectionParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/AppNameParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ApiVersionParameter"
+                    },
+                    {
+                        "name": "$skip",
+                        "in": "query",
+                        "required": false,
+                        "type": "integer",
+                        "description": "Control paging of export jobs, start results at the given offset, defaults to 0 (1st page of data).",
+                        "default": 0,
+                        "minimum": 0
+                    },
+                    {
+                        "name": "$top",
+                        "in": "query",
+                        "required": false,
+                        "type": "integer",
+                        "description": "Control paging of export jobs, number of export jobs to return with each call. By default, it returns all export jobs with a default paging of 20.\nThe response contains a `nextLink` property describing the path to get the next page if there are more results.\nThe maximum paging limit for $top is 40.",
+                        "default": 20,
+                        "maximum": 40,
+                        "minimum": 1
+                    },
+                    {
+                        "name": "$filter",
```

Comment:
This looks like a good candidate for OData extension - https://github.com/Azure/autorest/blob/master/Documentation/swagger-extensions.md#x-ms-odata